### PR TITLE
Refactor js code 

### PR DESCRIPTION
**Pull request by Marlyn-Mayienga**
In this module, we:
- renamed the branch refactor to refactor-js-code
- corrected the form to store the book data without refreshing the page
- modified the MIT.md

**NB**
This is a second review. We did the first review on the refactor branch (https://github.com/Marlyn-Mayienga/Awesome_Books/pull/2)  but had to rename it and it created a new PR.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -10,6 +10,10 @@
       integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC"
       crossorigin="anonymous"
     />
+    <link
```
Comment (arslanbisharat) [inline index.html:13]: - [ ] [OPTIONAL] Please make sure .vscode is added to the gitignore.

**Review by arslanbisharat — APPROVED** [review]
Hi @Marlyn-Mayienga,
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- The Pull Request has a proper title and description. ✔️ 
- You have added a descriptive Readme file. Good Job. ✔️ 
- There is no linter error is present. ✔️ 
- Project has met all the technical aspects you needed for this project. Keep rocking! ✔️ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by arslanbisharat** [comment]
Hi @Marlyn-Mayienga, when I started the review there was no linter issue. Please make sure there is no linter error before merging the branch.

**Comment by Marlyn-Mayienga** [comment]
> Hi @Marlyn-Mayienga, when I started the review there was no linter issue. Please make sure there is no linter error before merging the branch.

Hello,
Sure I will make sure all the tests pass the linter check. Thank you!

**Pull request merged**